vk/options.go

```diff
@@ -75,6 +76,11 @@ func (o *Options) ShouldUseHTTP() bool {
 
 // finalize "locks in" the options by overriding any existing options with the version from the environment, and setting the default logger if needed
 func (o *Options) finalize(prefix string) {
+	// Append trailing _ if prefix is missing one
+	if !strings.HasSuffix(prefix, "_") {
+		prefix = prefix + "_"
+	}
+
```

Comment:
We could describe this in the docs but I think we'd want people to use the prefix `APP_` over `APP`, where this fallback exists for backwards compatibility reasons.